.github/workflows/cpp-packaging.yml

```
@@ -699,7 +734,7 @@ jobs:
 
   merge_packages:
     name: final-merge-packages
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest-m
```

```suggestion
    runs-on: ubuntu-latest-m
```

Not enough disk space on default runner now, so using GitHub provided Large Runners here, but requires setup on each with a custom runner name (can all be done through GitHub UI)